code/datums/quirks/neutral_quirks/evil.dm

```diff
@@ -11,7 +11,23 @@
 	medical_record_text = "Patient has passed all our social fitness tests with flying colours, but had trouble on the empathy tests."
 	mail_goodies = list(/obj/item/food/grown/citrus/lemon)
 
+	/// Weak reference to the component handling our allergy to *love*
+	var/datum/weakref/added_allergies
+
+/datum/quirk/evil/add_unique(client/client_source)
+	var/mob/living/carbon/human/human_quirkholder = quirk_holder
+	var/obj/item/organ/heart/holder_heart = human_quirkholder.get_organ_slot(ORGAN_SLOT_HEART)
+	if(isnull(holder_heart) || isnull(holder_heart.reagents))
+		return
+	// Our actions have evaporated all the love that was once in our heart
+	holder_heart.reagents.del_reagent(/datum/reagent/love)
```

Comment:
should also shrink their heart, similarly to how friendly people have larger hearts

Reply:
how do you get this quirk really?